### ISSUE TITLE
Feature like "Indicator Workspaces"

### ISSUE THREAD

**Issue by adrianosaaquino** [elementary/gala#433]
I work with many static Workspace and is not possible.

A simples solutions: in settings, set number of Static Workspaces.

Workspace 1 -> Communication (email, messages)
Workspace 2 -> Project A Management
Workspace 3 -> Project A Development
Workspace 4 -> Project B Management
Workspace 5 -> Project B Development
Workspace 6 -> Empty
Workspace ......
Workspace 9 -> Paticular issues

**Comment by joe733** [comment]
Yes, something like optionally rename workspace, in the multitasking view, would be great for productivity
